components/esp_lvgl_port/README.md

```diff
@@ -99,6 +99,50 @@ Add touch input to the LVGL. It can be called more times for adding more touch i
 
 **Note:** The inputs added in this function are not removed in `lvgl_port_deinit`. They must be removed by `lvgl_port_remove_touch` before deinitialization. Otherwise, there can be memory leaks!
 
+### Add buttons input
+
+Add buttons input to the LVGL. It can be called more times for adding more buttons inputs for different displays. 
+``` c
+    /* Buttons configuration structure */
+    const button_config_t bsp_button_config[] = {
+        {
+            .type = BUTTON_TYPE_ADC,
+            .adc_button_config.adc_channel = ADC_CHANNEL_0, // ADC1 channel 0 is GPIO1
+            .adc_button_config.button_index = 0,
+            .adc_button_config.min = 2310, // middle is 2410mV
+            .adc_button_config.max = 2510
+        },
+        {
+            .type = BUTTON_TYPE_ADC,
+            .adc_button_config.adc_channel = ADC_CHANNEL_0, // ADC1 channel 0 is GPIO1
+            .adc_button_config.button_index = 1,
+            .adc_button_config.min = 1880, // middle is 1980mV
+            .adc_button_config.max = 2080
+        },
+        {
+            .type = BUTTON_TYPE_ADC,
+            .adc_button_config.adc_channel = ADC_CHANNEL_0, // ADC1 channel 0 is GPIO1
+            .adc_button_config.button_index = 2,
+            .adc_button_config.min = 720, // middle is 820mV
+            .adc_button_config.max = 920
+        },
+    };
+
+    const lvgl_port_nav_btns_cfg_t btns = {
+        .disp = disp_spi,
+        .button_prev = &bsp_button_config[0],
+        .button_next = &bsp_button_config[1],
+        .button_enter = &bsp_button_config[2]
+    };
+
+    /* Add buttons input (for selected screen) */
+    lvgl_port_add_navigation_buttons(&btns);
+```
+
+**Note 1:** The inputs added in this function are not removed in `lvgl_port_deinit`. They must be removed by `lvgl_port_remove_navigation_buttons` before deinitialization. Otherwise, there can be memory leaks!
```

Comment:
Optional suggestion: can we convert this note into a part of the code snippet above? For example:

```c++
    /* Add buttons input (for selected screen) */
    lv_indev_t* buttons_handle = lvgl_port_add_navigation_buttons(&btns);
    
    /* ... the rest of the initialization ... */

    /* If deinitializing LVGL port, remember to delete the buttons: */
    lvgl_port_remove_navigation_buttons(buttons_handle);
```

Reply:
Great! Thank you! Added to all parts (touch, display) instead of notes.